openedx_lti_tool_plugin/waffle.py

```diff
@@ -0,0 +1,9 @@
+
+"""Waffle flags for openedx_lti_tool_plugin."""
+from edx_toggles.toggles import WaffleSwitch
+
+from openedx_lti_tool_plugin.apps import OpenEdxLtiToolPluginConfig as AppConfig
+
+COURSE_ACCESS_CONFIGURATION = WaffleSwitch(
+    f'{AppConfig.name}.course_access_configuration', __name__
+)
```

Comment:
```suggestion

WAFFLE_NAMESPACE = AppConfig.name

COURSE_ACCESS_CONFIGURATION = WaffleSwitch(
    f'{WAFFLE_NAMESPACE}.course_access_configuration', __name__
)
```

What do you think of adding this constant to be more explicit?